src/librawrtc/sctp_transport.c

```diff
@@ -1,8 +1,11 @@
 #include <stdio.h> // fopen
 #include <string.h> // memcpy, strlen
 #include <errno.h> // errno
+#ifndef _WIN32
 #include <sys/socket.h> // AF_INET, SOCK_STREAM, linger
 #include <netinet/in.h> // IPPROTO_UDP, IPPROTO_TCP, htons
+#endif
+#include <pthread.h>
```

Comment:
The source file doesn't use any `pthread` functionality. If this is required, it's likely a missing include in usrsctp. Can you paste the compile error?